#reason: fish-shell requires 3rd-party completion files to be installed to vendor_completions.d

--- setup.py
+++ setup.py
@@ -119,7 +119,7 @@
       data_files = [('man/man1', ['0launch.1', '0alias.1', '0store-secure-add.1', '0store.1', '0desktop.1', '0install.1']),
 		    ('share/applications', ['share/applications/0install.desktop']),
 		    ('share/bash-completion/completions', ['share/bash-completion/completions/0install']),
-		    ('share/fish/completions', ['share/fish/completions/0install.fish']),
+		    ('share/fish/vendor_completions.d', ['share/fish/completions/0install.fish']),
 		    (zsh_functions_dir, ['share/zsh/site-functions/_0install']),
 		    ('share/icons/hicolor/24x24/apps', ['share/icons/24x24/zeroinstall.png']),
 		    ('share/icons/hicolor/48x48/apps', ['share/icons/48x48/zeroinstall.png']),
